PyOpenWorm/dataObject.py

```diff
@@ -1,432 +1,223 @@
+from __future__ import print_function
 import rdflib as R
-from .data import DataUser
-from .configure import BadConf
-import itertools as IT
-import traceback
+import random as RND
 import logging
 
-__all__ = ["DataObject", "Property", "SimpleProperty", "values"]
-L = logging.getLogger(__name__)
-
-# in general it should be possible to recover the entire object from its identifier: the object should be representable as a connected graph.
-# However, this need not be a connected *RDF* graph. Indeed, graph literals may hold information which can yield triples which are not
-# connected by an actual node
-
-def _bnode_to_var(x):
-    return "?" + x
+from yarom.graphObject import GraphObject, ComponentTripler, GraphObjectQuerier
+from yarom.rdfUtils import triples_to_bgp, deserialize_rdflib_term
+from yarom.rdfTypeResolver import RDFTypeResolver
```

Comment:
I would prefer to add documentation to YAROM than to duplicate code between these two projects. Is that alternative acceptable?